Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -488,10 +488,21 @@ public override Ust VisitObjectCreationExpression(ObjectCreationExpressionSyntax
 
         public override Ust VisitInitializerExpression(InitializerExpressionSyntax node)
         {
+            Ust result = null;
             var children = node.Expressions.Select(e => (Expression)VisitAndReturnNullIfError(e))
                 .ToArray();
-            
-            var result = new MultichildExpression(children, node.GetTextSpan());
+            var textSpan = node.GetTextSpan();
+
+            if (node.Kind() == SyntaxKind.ArrayInitializerExpression)
+            {
+                var sizes = new List<Expression> { new IntLiteral(children.Length) };
```

Comment:
2D, 3D and so on array initializers are impossible?

Reply:
In this kind of initializer i decide to represent 2d as 1d of 1d arrays and etc. I do this in order not to collect information from child arrays